core/src/main/java/com/homeaway/streamplatform/streamregistry/dto/AvroToJsonDTO.java

```diff
@@ -27,15 +27,7 @@
 
 import com.homeaway.digitalplatform.streamregistry.AvroStream;
 import com.homeaway.digitalplatform.streamregistry.RegionStreamConfiguration;
-import com.homeaway.streamplatform.streamregistry.model.ClusterKey;
-import com.homeaway.streamplatform.streamregistry.model.ClusterValue;
-import com.homeaway.streamplatform.streamregistry.model.Consumer;
-import com.homeaway.streamplatform.streamregistry.model.Producer;
-import com.homeaway.streamplatform.streamregistry.model.RegionStreamConfig;
-import com.homeaway.streamplatform.streamregistry.model.Schema;
-import com.homeaway.streamplatform.streamregistry.model.Stream;
-import com.homeaway.streamplatform.streamregistry.model.StreamConfig;
-import com.homeaway.streamplatform.streamregistry.model.Tags;
+import com.homeaway.streamplatform.streamregistry.model.*;
```

Comment:
Please un-star the imports

Reply:
marking unresolved until ... fixed (e.g. resolved)